git_tools/create_opence_release.py

```diff
@@ -55,9 +61,14 @@ def _main(arg_strings=None):
     version_name = "open-ce-v{}".format(args.version)
     release_number = ".".join(args.version.split(".")[:-1])
     branch_name = "open-ce-r{}".format(release_number)
-    version_msg = "Open-CE Version {}".format(args.version)
     primary_repo_url = "git@github.com:{}/{}.git".format(args.github_org, args.primary_repo)
 
+    version_msg = "Open-CE Version {}".format(args.version)
+    release_name = "v{}".format(args.version)
+    if args.code_name:
+        version_msg = "{} Condenamed {}".format(version_msg, args.code_name)
```

Comment:
ha. I thought this said "condemned" at first sight. Maybe it needs a dash?

Reply:
Haha, I copied Ubuntu's wording for their release code names.
Apparently it needs a dash as a verb: https://whatis.techtarget.com/feature/Which-is-correct-codenamed-or-code-named